kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/KafkaServiceReconciler.java

```diff
@@ -123,6 +152,24 @@ static PrimaryToSecondaryMapper<KafkaService> kafkaServiceToConfigMap() {
                 .orElse(Set.of());
     }
 
+    @VisibleForTesting
+    static SecondaryToPrimaryMapper<Kafka> kafkaToKafkaService(EventSourceContext<KafkaService> context) {
```

Comment:
nit:  lets make the code clear that we are talking about Strimzi CRs.  so kafkaToKafkaService ->  strimziKafkaToKafkaService